content/2022-01-04-brigade-2.2.0.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "Brigade 2.2.0"
+slug: "2022-01-04-brigade-2.2.0"
+description: "Brigade 2.2.0 has been released"
+date: "2022-01-04"
+author: "Kent Rancourt"
+summary: "Brigade 2.2.0 has been released."
+---
+
+Happy New Year, Brigadiers!
+
+We're starting the new year off right with our second minor release of Brigade 2.
+
+v2.2.0 continues the trend started with v2.1.0 by delivering just a handful of small, incremental improvements. Notably, in this release, we've corrected a bug in the user list endpoint and made it easy for script authors to designate jobs as fallible -- that is, _allowed to fail without forcing the overall workflow defined by the script to fail_. This is a great feature for cases where script authors may wish to treat a certain job's failure as informational in nature rather than fatal.
+
+To demonstrate, fallible jobs, consider this small fragment of a `brigade.ts` file:
```

Comment:
`To demonstrate, fallible jobs` should be `To demonstrate fallible jobs`

Reply:
Not sure how I botched that. Fixing!

Reply:
Fixed in 73f0cf5b716271193dea9b37dab40fd2a7151dad